frontend/src/components/HospitalAdmin.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Admin, Resource} from 'react-admin';
+import jsonServerProvider from  'ra-data-json-server';
+import InventoryList from './InventoryList';
+import UserList from './UserList';
+import HospitalAdminLandingPage from './HospitalAdminLandingPage';
+import HospitalAddressShow from './HospitalAddressShow';
+import HospitalAddressEdit from './HospitalAddressEdit';
+import UserEdit from './UserEdit';
+import InventoryEdit from './InventoryEdit';
+
+export default () => {
+
+  // const dataProvider = jsonServerProvider('https://jsonplaceholder.typicode.com');
+  const dataProvider = jsonServerProvider('http://my-json-server.typicode.com/naijauser/infomedapi');
+
+  return (
+    <div>
+      <Admin dashboard={HospitalAdminLandingPage} dataProvider={dataProvider} > 
```

Comment:
There's a problem with React router: it doesn't properly resolve params inside. See https://marmelab.com/react-admin/CustomApp.html for alternatives.